app/views/claims/schools/users/index.html.erb

```diff
@@ -4,4 +4,30 @@
 
 <div class="govuk-width-container">
   <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= govuk_button_to(t(".add_user"), new_claims_school_user_path, method: :get) %>
+
+      <% if @users.any? %>
+        <table class="govuk-table">
```

Comment:
Could we use the govuk_table component? See [here](https://github.com/DFE-Digital/itt-mentor-services/blob/d8b74634014de56998d6cf585466844257435610/app/views/placements/support/schools/users/index.html.erb)
Again, it's now something that is used in many places, so good time to make it a shared component. And in doing so, you'll fix the empty state problem everywhere at once.